### DISCUSSION TITLE
What is the smallest ocean in the world 🌊 ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#40] [Galaxy Qs] [answered]
What is the smallest ocean 🌊 in the world ?

**Comment by zolagonano** [comment, marked as answer]
The smallest ocean in the world is the Arctic Ocean.

**Reply by Sriansh-raj** [reply]
Congratulations @zolagonano you just got a `Galaxy Brain` achievement. 🎉 

Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃
